ui/src/app/base/components/FormikFormButtons/FormikFormButtons.test.tsx

```diff
@@ -71,7 +71,36 @@ describe("FormikFormButtons ", () => {
         <FormikFormButtons buttonsBordered submitLabel="Save" />
       </Formik>
     );
-    expect(wrapper.find("hr").exists()).toBe(true);
+    expect(
+      wrapper
+        .find("[data-test='buttons-wrapper']")
+        .prop("className")
+        ?.includes("is-bordered")
+    ).toBe(true);
+  });
+
+  it("displays inline if inline is true", () => {
+    const wrapper = mount(
+      <Formik initialValues={{}} onSubmit={jest.fn()}>
+        <FormikFormButtons inline submitLabel="Save" />
+      </Formik>
+    );
+    expect(
+      wrapper
+        .find("[data-test='buttons-wrapper']")
+        .prop("className")
+        ?.includes("is-inline")
+    ).toBe(true);
+  });
+
+  it("displays help text if provided", () => {
+    const buttonsHelp = <p>Help!</p>;
+    const wrapper = mount(
+      <Formik initialValues={{}} onSubmit={jest.fn()}>
+        <FormikFormButtons buttonsHelp={buttonsHelp} submitLabel="Save" />
+      </Formik>
+    );
+    expect(wrapper.find("[data-test='buttons-help']").exists()).toBe(true);
```

Comment:
You could check the contents here which would make the test clearer?

Reply:
Good call, [done](https://github.com/canonical-web-and-design/maas-ui/pull/2949/files#diff-89b078473e1c25340f867defaf98bd294acc265bd61fceae27309fa26e8063a2R104).